index.html

```diff
@@ -102,13 +102,28 @@ <h2 id="document_purpose">Purpose of this document</h2>
   and because those designing such system may wish to provide alternative options.
   Note that the terminology is based on that defined in [[JLREQ]]. </p>
 
-<aside class="note" title="Ruby and interlinear notes" id="n20200529001"> Interlinear notes are sometimes placed between lines
-      in the same position as ruby annotations
-      (see also JLReq <a href="https://www.w3.org/TR/jlreq/#fig3_2_8">Figure 241: An example of a note in inter lines</a> for examples).
-      These inter-linear notes are not covered in this document.
-      Line breaks are not allowed inside mono-ruby annotations, and, for group-ruby, line-breaks are not allowed in the ruby base. 
-      In contrast, interlinear notes may involve a large number of characters, 
-      and it  must be possible to break lines within both the base text and the note itself, and use different rules for placement. </aside>
+<aside class="note" title="Ruby and interlinear notes" id="n20200529001">
+<p>Interlinear notes are sometimes placed between lines in the same position as ruby annotations
+(see also JLReq <a href="https://www.w3.org/TR/jlreq/#fig3_2_8">Figure 241: An example of a note in inter lines</a> for examples).
```

Comment:
I am worried that the figure number will change in a new jlreq version. Can we remove "Figure 241"?

Reply:
thank you! fixed.

Reply:
Hmm, I don't know if it is a caching problem or you fixed it in another branch... I still see it here.

Reply:
sorry, was local conflict during push (forgot to pull before updating)